plugins/simpleadminhacks-paper/src/main/java/com/programmerdan/minecraft/simpleadminhacks/framework/utilities/BuildLimit.java

```diff
@@ -0,0 +1,81 @@
+package com.programmerdan.minecraft.simpleadminhacks.framework.utilities;
+
+import org.bukkit.configuration.serialization.ConfigurationSerializable;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class BuildLimit implements ConfigurationSerializable {
+	private String world;
+	private String type;
+	private int min_y;
+	private int max_y;
+
+	public BuildLimit (String world, String type, int min_y, int max_y){
+		this.world = world;
+		this.type = type;
+		this.min_y = min_y;
+		this.max_y = max_y;
+	}
+
+	public String getWorld() {
+		return this.world;
+	}
+	public String getType() {
+		return this.type;
+	}
+	public int getMin_y() {
+		return this.min_y;
+	}
+	public int getMax_y() {
+		return this.max_y;
+	}
+
+	public void setWorld(String world) {
+		this.world = world;
+	}
+	public void setType(String type) {
+		this.type = type;
+	}
+	public void setMin_y(int min_y) {
```

Comment:
`setMin_y` ?

Reply:
What's wrong?

Reply:
Just seems like an odd naming convention to have both camel and snake case. It's typical for getters in Java to use camel casing (eg: `setMinY`) or otherwise to have the field public if you want to retain the name.

Reply:
Oh, I used `y_min` in the config so it just got carried over to the var names. Now that I look at it, it does feel a bit weird

Reply:
Yeah `setMinY` is correct, you never wanna make exceptions to name casing.